pkg/operator/controller/gatewayclass/subscription.go

```diff
@@ -16,6 +17,10 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
+const (
+	serviceMeshOperatorNamespace = "openshift-operators"
```

Comment:
This namespace should be declared a global default as mentioned in https://github.com/openshift/cluster-ingress-operator/pull/1112/files?diff=split&w=1#r1958739919.